covid19count.py

```diff
@@ -192,7 +232,13 @@ def fatality(regions: List[str]):
     plt.xlabel("")
     plt.ylim(0)
     _draw_watermark(plt.gcf())
-    plt.show()
+
+    if ( outfile is not None and len(outfile) > 0 ):
```

Comment:
No need to check if outfile is None, just do `if outfile:`

Reply:
Me and Python go hand in hand as you can see.

Reply:
I C you've been writing C :wink: